Sources/MongoKitten/AggregateBuilder.swift

```diff
@@ -0,0 +1,154 @@
+import BSON
+
+
+/// An object that can be used to build an aggregate command
```

Comment:
It's not an aggregate command, right? It's a builder for nested pipelines specifically.